### DISCUSSION TITLE
The > question about :has

### DISCUSSION THREAD

**Discussion by CaiSijian** [ankit/stylebot#739] [General]
我是一位来自中国的朋友
是这样的，我尝试在has中使用>表示子级选择。但是stylebot就报错了。例子如下：
 `
:has(>#one)>:not(.one) {
       color: red;
}
`
我通过vscode进行尝试，上面的代码是有效的，但是在stylebot就报错了，这是一个bug，希望修复。
感谢你。

I am a friend from China
Well, I try to use>to represent sub level selection in has. But the stylebot reported an error. Examples are as follows:
`
:has(>#one)>:not(.one) {
color: red;
}
`
I tried through vscode. The above code is valid, but an error is reported in the stylebot. This is a bug and I hope to fix it.
Thank you.

**Comment by generrosity** [comment]
Stylebot uses a 'CSS Lint' module to show the red errors  - its that module needs to be updated to understand the latest CSS
